module-infrastructure/persistence-db/src/main/kotlin/site/yourevents/guest/repository/GuestRepository.kt

```diff
@@ -25,4 +27,19 @@ class GuestRepository(
         return guestJPARepository.findById(guestId)
             .getOrNull()?.toDomain()
     }
+
+    override fun getReceivedInvitationCount(member: Member): Int {
+        return MemberEntity.from(member)
+            .let { memberEntity ->
+                guestJPARepository.getReceivedInvitationCount(memberEntity)
+            }
+    }
+
+    override fun getGuestsOfReceivedInvitation(member: Member): List<Guest> {
+        return MemberEntity.from(member)
+            .let { memberEntity ->
+                guestJPARepository.getGuestsOfReceivedInvitation(memberEntity)
+                    .map { it.toDomain() }
+            }
+    }
```

Comment:
let을 사용하지 않음으로서 행위를 더욱 명확히 표현해줄 수 있다고 생각합니다! 👍 
```suggestion
    override fun getReceivedInvitationCount(member: Member): Int =
        guestJPARepository.getReceivedInvitationCount(
            MemberEntity.from(member))

    override fun getGuestsOfReceivedInvitation(member: Member): List<Guest> =
        guestJPARepository.getGuestsOfReceivedInvitation(
            MemberEntity.from(member))
                .map { it.toDomain() }
```

Reply:
수정했습니다!